packages/component-utils/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@packages/component-utils",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
```suggestion
  "private": false,
```

believe this needs to be false for npm to publish

Reply:
I don't know about that -- monorepos are different I think. I'll test it out